e2e/app/geth/config_internal_test.go

```diff
@@ -103,7 +103,8 @@ func gethDumpConfigToml(t *testing.T, baseCfg FullConfig) []byte {
 		fmt.Sprintf("--volume=%s:/tmp/config.toml", baseFile),
 		fmt.Sprintf("ethereum/client-go:%s", Version),
 		"dumpconfig",
-		"--config=/tmp/config.toml")
+		"--config=/tmp/config.toml",
+		"--discv5=false")
```

Comment:
the point of this test it to see what geth does with provided config. If it enables discv5 always, no matter what we provide in the config, then it will also do that in prod. So best make it explicit that discv5 is enabled in that case.

So please remove this line, enable discv5 in `MakeGethConfig` and update golden files to indicate that discv5 is enabled. Since it is very hard to disable it.